src/server/hx/map.rs

```diff
@@ -0,0 +1,35 @@
+use crate::db;
+use crate::server::Config;
+
+use super::render::ServerError;
+use super::render::{Response, TemplatedResponse};
+use super::Media;
+use super::State;
+use actix_web::get;
+use actix_web::web::Data;
+use actix_web::web::Query;
+use askama::Template;
+use sqlx::Pool;
+use sqlx::Sqlite;
+
+#[derive(Template)]
+#[template(path = "web/map/map.html")]
+struct HxMap<'a> {
+    media: &'a Vec<Media>,
+}
+
+pub(super) async fn render(pool: &Pool<Sqlite>, config: &Config, state: State) -> Response {
+    let media = db::media_with_pos(pool)
```

Comment:
```suggestion
    let media = db::media_with_pos(pool, &state)
```

Pass in state to filter ....